packages/yoroi-ergo-connector/src/background.js

```diff
@@ -1,6 +1,4 @@
 
-
 chrome.browserAction.onClicked.addListener(function() {
-    const yoroiExtensionId = "bgihpbbhciffmelcfbccneidnnmkcdhl";
     chrome.runtime.sendMessage(yoroiExtensionId, { type: "open_browseraction_menu" });
```

Comment:
I'm getting `extensionId` generated for this file (+inject.js) but it's still referencing it as `yoroiExtensionId`